idb/postgres/postgres.go

```diff
@@ -1255,15 +1255,12 @@ ON CONFLICT (addr, assetid) DO UPDATE SET amount = account_asset.amount + EXCLUD
 	staterow := tx.QueryRow(`SELECT v FROM metastate WHERE k = 'state'`)
 	var stateJSONStr string
 	err = staterow.Scan(&stateJSONStr)
-	if err == sql.ErrNoRows {
```

Comment:
If we're going to make changes here, maybe we should add `GetImportState` and `SetImportState` back. Then the error would be the `ErrorNotInitialized` error.

Reply:
Yes, I'm going to do it in another PR.